pom.xml

```diff
@@ -94,6 +260,21 @@
               <includeArtifactIds>ozone-distro</includeArtifactIds>
             </configuration>
           </execution>
+
+          <execution>
+            <id>Copy Ozone Docker Embedded Files</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>unpack-dependencies</goal>
+            </goals>
+            <configuration>
+              <excludeTransitive>true</excludeTransitive>
+              <outputDirectory>
+                ${project.build.directory}/${project.artifactId}-${project.version}/docker-embedded</outputDirectory>
```

Comment:
I'm not a fan at all to end up with `distro/` `run/` and `docker-embedded/` at the same level.
I almost would like this to be a separate artifact altogether. Is this project actually needed at runtime?

Reply:
The project is not needed at runtime. I exclude it from the final archive
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/a1facb5b-b626-4666-be31-7c927af043cb">
[see](https://github.com/ozone-his/ozone/blob/1b989fb0b2a0b083f65d8c5bd1c2acec295547a0/assembly.xml#L17)

Reply:
> I almost would like this to be a separate artifact altogether. Is this project actually needed at runtime?


I thought about whether the project should be a separate artifact, but it could complicate the building of images because of the build context. The docker-embedded project and the distro need to sit next to each other. This is not the case every time, but we have [cases](https://github.com/ozone-his/ozone/blob/1b989fb0b2a0b083f65d8c5bd1c2acec295547a0/pom.xml#L120) where it becomes necessary.

Reply:
What makes the frontend image build above different from the others is that it needs files from both projects. In this case on top of the distro it needs the custom [startup.sh](https://github.com/ozone-his/ozone-docker-embedded/blob/24be85e8e3c8c835306a393c8fa30a2b6896638f/docker/frontend/startup.sh) from the docker-embedded project